### PR TITLE
Project prototype.

### PR DESCRIPTION
**Pull request by AlexandruGH**
Greetings! I have some problems with the project in the gui part.I traced the program with the debugger and i noticed that even if my program runs the canvas that should display the Huffman Tree,it doesn't display anything.When I skip to the last method that is called canvas.display() ,it seems that the program enters an infinite loop.Overall the algorithm works.Should I use a different approach in the gui part?

### CODE REVIEW

File: MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/Gui.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+						JOptionPane.showMessageDialog(Gui.this, "Please enter some text.", "Error",
+								JOptionPane.ERROR_MESSAGE);
+						System.exit(0);
+					}
+					huffman = new Huffman(inputText);
+					huffman.determineNodePositions();
+					huffman.determineMaximumHeight();
+					canvas.clear();
+					drawTree(huffman.getRootOfHuffmanTree());
+					canvas.display();
```
Comment (AlexandruGH) [inline MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/Gui.java]: Here is the part that doesn't work.

File: MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/Gui.java [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+		codePanel.add(codeDisplay);
+		this.add(codePanel, BorderLayout.EAST);
+		this.getContentPane().add(canvas, BorderLayout.WEST);
+		this.setVisible(true);
+
+	}
+
+	// WEST SECTION
+	// Inorder traversal
+	public void drawTree(Node root) {
```
Comment (AlexandruGH) [inline MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/Gui.java]: Algorithm seems to run well ,but it doesn't display anything.

**Comment by Alex-Cosma** [comment]
Please be careful on where you create this 'repository'. You created a separate folder outside the actual `Students` folder. Make sure to repair it by removing this folder from there and moving it in the right place.

**Comment by Alex-Cosma** [comment]
Also, if you used jgraphx, make sure to add the library as well

**Comment by Alex-Cosma** [comment]
Oh you've only imported the library but didn't use it. You know what, `jgraphx` is a great library and I think it would be a great idea to use it to plot your trees, what do you think?

**Comment by AlexandruGH** [comment]
Greetings! Firstly I don't really figure out how to change the destination of the folder.This happened because I had some problems these days with github.I also tought of using jgraphx or javafx for displaying the binary tree.

**Comment by Alex-Cosma** [comment]
Ok so you can simply delete your current repository, and for a new one. Then, just move your project in the `\Students\30425\Ghiurutan V. Alexandru\` folder.

**Comment by Alex-Cosma** [comment]
I have used jgraphx before. It's quite easy to use and nice. I recommend it! :+1:

**Pull request closed without merging**